content/docs/1.18/security.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Securing Jaeger Installation
+hasparent: true
+---
+
+This page documents the existing security mechanisms in Jaeger, organized by the pairwise connections between Jaeger components. We ask for community help with implementing additional security measures (see [issue-1718][]).
+
+## Client to Agent
+
+Deployments that involve Jaeger Agent are meant for trusted environments where the agent is run as a sidecar within the container's network namespace, or as a host agent. Therefore, there is currently no support for traffic encryption between clients and agents.
+
+* [ ] Sending trace data over UDP - no TLS/authentication.
+* [ ] Retrieving sampling configuration via HTTP - no TLS/authentication.
+
+## Client to Collector
+
+Clients can be configured to communicate directly with the Collector via HTTP. Unfortunately, at this time the Jaeger backend does not provide means of configuring TLS for its HTTP servers. The connections can be secured by using a reverse proxy placed in front of the collectors.
+
+* [ ] HTTP - no TLS/authentication.
+  * Some Jaeger clients support passing auth-tokens or basic auth.
```

Comment:
We could add a link to client features page.

Reply:
added the link, but the anchor doesn't work great with a table with sticky header